packages/icm20948-1.0.0.yaml

```diff
@@ -0,0 +1,5 @@
+name: icm20948
```

Comment:
Either you need to change the title in the README, or you need to add a name in the package.yaml file.
The readme is currently saying "icm20948-driver".